src/Controller/ProductListController.php

```diff
@@ -0,0 +1,45 @@
+<?php 
+
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Entity\Product;
+use App\Entity\Category;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/***
+ * 
+ * Controller wich provides the products list
+ * 
+ */
+class ProductListController{
+
+    public function __invoke(Request $request, Container $container){
+
+        $entityManager = $container->get('entity.manager');
+        $entityManager->resetDatabase();
+        $category = new Category();
+        $category->setName("Burger");
+
+        $product = new Product();
+        
+        $product->setName("Big mac");
+        $product->setPrice(7.95);
```

Comment:
you should never rely on float values especially for prices.
always store an integer containing 3 digits after the separation. like: 1000 to store 1€
for currencies conversion it's a mess and to know why try this in your console:

```shell
$ php -a
php > var_dump(0.1+0.2 === 0.3);
```
```